mysql-test/lib/mtr_report.pm

```diff
@@ -36,6 +36,8 @@ use mtr_match;
 use My::Platform;
 use POSIX qw[ _exit ];
 use IO::Handle qw[ flush ];
+use mtr_result_names qw(MTR_RES_FAILED MTR_RES_PASSED MTR_RES_SKIPPED
```

Comment:
@timofeyturenko  my opinion is that there is no need to add additional module just to have constants used for some part of reporting there. One can still compare strings and if needed adding comment about the meaning of the constants in report. This will leave the patch with only meaningful changes which are easier to review. 
So I would remove this everywhere in the patch.

Reply:
`MTR_RES` constants are used in several files. It will not be very good if someone makes a mistake in a string constant